docs/index.md

```diff
@@ -12,7 +12,54 @@ The PingFederate provider manages the configuration of a PingFederate server.
 
 The PingFederate provider supports version `11.2` of PingFederate.
 
-<!-- TODO add example usage -->
+## Documentation
+Detailed documentation on PingFederate can be found in the [online docs](https://docs.pingidentity.com/r/en-us/pingfederate-112/pf_pingfederate_landing_page)
+### Example Usage of PingFederate Provider
+```terraform
+terraform {
+  required_version = ">=1.1"
+  required_providers {
+    pingfederate = {
+      version = "~> 0.1.0"
+      source  = "pingidentity/pingfederate"
+    }
+  }
+}
+
+provider "pingfederate" {
+  username                            = "administrator"
+  password                            = "2FederateM0re"
+  https_host                          = "https://localhost:9999"
+  insecure_trust_all_tls              = true
+  x_bypass_external_validation_header = true
+}
+
+resource "pingfederate_administrative_account" "myAdministrativeAccount" {
+  username    = "example"
+  description = "description"
+  password    = "2FederateM0re"
+  roles       = ["USER_ADMINISTRATOR"]
+}
+```
+### Using resources that have a different type
```

Comment:
I think "have a different type" is kind of unclear language. Maybe something more like "resources that support multiple distinct types".